enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.ts

```diff
@@ -59,6 +59,11 @@ export function getEmbeddingThemeOverride(
         const themeColorNames =
           SDK_TO_MAIN_APP_COLORS_MAPPING[name as MappableSdkColor];
 
+        // If the sdk color does not exist in the mapping, skip it.
+        if (!themeColorNames) {
+          continue;
+        }
```

Comment:
The SDK throws a JavaScript iterable error if you pass in a non-existent color name. Best to ignore it. I was thinking of adding a `console.warn` here but I'm afraid that would get spammy.

Reply:
hmm. I think swallowing errors isn't a good idea, I'd rather be thrown errors in the face than not knowing what I did wrong.

Reply:
@WiNloSt Good point, let me add a console.warn message there so they're aware of the issue, but it doesn't crash their app with weird `Iterable` errors.

Reply:
Added a warning message in https://github.com/metabase/metabase/pull/50299/commits/ac561efe8728580c439476757ebd4a73c2eeec0c